specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,2898 @@
+            "schema": {
+              "$ref": "#/definitions/ProjectDeletionJobStatusObject"
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "404": {
+            "description": "The operation does not exist."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      }
+    },
+    "/projects/global/languages": {
+      "get": {
+        "description": "Get List of Supported languages.",
+        "operationId": "Projects_GetSupportedCultures",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "projectKind",
+            "description": "The project kind to get the supported languages for.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "conversation",
+              "workflow"
+            ]
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of supported languages.",
+            "schema": {
+              "$ref": "#/definitions/LanguagesGetObject"
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "404": {
+            "description": "The project does not exist."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      }
+    },
+    "/projects/global/prebuiltEntities": {
```

Comment:
>E [](http://example.com/codeflow?start=30&length=1)

inconsistent spelling. There are locations separated with - like deletion-jobs and here there is casing used (where it will be all lower cased anyway). Should this be prebuilt-entities?